### ISSUE TITLE
Add a tutorial for novice user

### ISSUE THREAD

**Issue by sharkAndshark** [urbica/martin#1254]
Add a tutorial which contains whole steps of starting a map tiles service with demo data

**Comment by CommanderStorm** [comment]
I think this is resolved, or are we missing an aspect? => https://maplibre.org/martin/recipe-basemap-postgis.html

**Comment by sharkAndshark** [comment]
Yeah And the [quick start](https://maplibre.org/martin/quick-start.html)
